gui/src/components/health/cortx-health.vue

```diff
@@ -15,15 +15,61 @@
 * please email opensource@seagate.com or cortx-questions@seagate.com.
 */
 <template>
-  <router-view class="cortx-p-1"></router-view>
+  <div class="cortx-p-1">
+    <cortx-tabs :tabsInfo="tabsInfo" />
+      <CortxHealthGraphical v-if="showGraphTab" />
```

Comment:
Please remove graphical tab as it is not implemented yet.

Reply:
Removed Graph tab